### DISCUSSION TITLE
Transparent background is not supported when displaying the logo

### DISCUSSION THREAD

**Discussion by ot-i-to** [requarks/wiki#6417] [Error / Bug Report] [answered]
Wiki.js 2.5.298
Node.js 16.20.0
SQLite 3.40.0 
Linux - Ubuntu (jammy) 22.04 x64

I ran into this when using a PNG file in a logo :(

**Comment by NGPixel** [comment, marked as answer]
Transparent PNGs are supported... Make sure your image is actually transparent?
